engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -221,6 +233,19 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
       log.info(s"$tag transitioning from $fromState to $toState.")
   }
 
+  private def handleRetryableFailure(jobKey: BackendJobDescriptorKey) = {
```

Comment:
I would defer to @Horneth as the author of preemptibility, but I thought the original key would get status "preempted" and a new key would be created with status "not started".